boards/arm/v2m_musca_b1/v2m_musca_b1_nonsecure.dts

```diff
@@ -42,14 +42,14 @@
 		};
 	};
 
-	flash0: flash@0a060400 {
+	flash0: flash@0a070400 {
```

Comment:
The commit that changes this is for psa tests, do this in as `samples/tfm_integration/tfm_psa_arch_tests/boards/v2m_musca_b1.overlay`

Reply:
TF-M has also gotten larger with new features since the last 1.0rc2 commit, and I would propose just leaving the extra 64KB to allow room for future changes to the S image and bootloader?